joke-data-crawler/src/chucknorrisapi.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+url = "https://api.chucknorris.io/jokes/random"
+
+
+def load_jokes():
```

Comment:
```suggestion
def load_joke():
```
i would rename the function. It only returns one joke